### DISCUSSION TITLE
Hints on what is wrong ?

### DISCUSSION THREAD

**Discussion by Aponton** [frntc/RAD#26] [Q&A] [answered]
Hello from Brussels, 
I've built the RAD, sourced all the IC's from mouser.com and got some wierd image on the C64:
![IMG_6907](https://github.com/frntc/RAD/assets/4056792/b460317f-fff0-427c-9804-85a3f97bac26)
The music is playing and pushing the "H" letter changes the image on the screen but still unreadable lines...
I've tried several firmwares with the same results.

more info:
Raspberry is externaly feeded (no jumper).
Raspberry Pi 3 Model B (AArch64) - speed factor is 3.57 valid version 3.0x SD card (on Rapsberry HDMI display)
C64C  - ASSY 250469 PCB 252311 rev A
C64 boots fine to standard basic when RAD not inserted.
Other cartridges works fine KungFu Flash / EasyFlasy / FinalCartridge III +

Thanks in advance for your help :)

**Comment by frntc** [comment]
Hi to Brussels!

Is this an RPi 3B or 3B+? The former exists with two steppings of the SoC and the older one is probably not suited. What's the exact SoC on it?

**Reply by Aponton** [reply]
Hello frntc, 
thanks for the reply,
the Raspberry is a 3B V1.2 (no "+") from 2015 and the SoC is a Broadcom BCM2837RIFBG - TN1545 - P11 
I've found and tested with another RP 3B +,  it has the same symptoms the SoC is BCM2837B0IFSBG - TA1748 - P20
Does the project run on RP4 model B ?
Thanks again for your project, time and your kind response :)

**Comment by frntc** [comment, marked as answer]
No it doesn't work with a RP4. You need a 3A+, 3B+ or Zero 2. Unfortunately the SoC of your 3B is the old one...

**Reply by Aponton** [reply]
Hello frntc,
just an update to point that the problem was a bad solder joint on the lvc573 near the 40 pin connector.
I've also bought 2 RPI 3A+, tested and it works.
Just for fun, i also tried with my 3B and 3B+, and it also works perfect !
Thanks again for your help !
Tony

**Reply by frntc** [reply]
thanks for the update!

**Comment by robxbl69** [comment]
Just wanted to say a big thanks to frntc for all his amazing work and to Aponton for starting this thread as I was having similar issues. Tried 4 different 64's and all I got was gibberish and corruption on the screen. 

I purchased a Sidekick64 and RAD from retro=store.de back in July and only now the weather has turned wet and horrible have I got round to trying them :-/ 

The Sidekick64 worked first time (other than the TFT screen as I hadn't blobbed the pads to configure the +/= to the display) however the RAD was proving to me more challenging. After messing with timings and trying other 64's (both breadbin and short board C's) I was about to give up, but then found this thread and after inspecting the PCB under magnification I found that one leg was not soldered on one of the IC's near the GPIO connector.

So now off to try Sonic... 

Thanks, Rob

**Reply by frntc** [reply]
Hi,

thanks for the feedback! The SK64s and RADs from restore-store are typically manufactured with SMD-parts and problems should not occur -- apparently your RAD shows the opposite. If something doesn't work, Konstantin from restore-store is very helpful, just contact him!
